hs_core/views/__init__.py

```diff
@@ -1029,20 +1030,18 @@ def create_resource(request, *args, **kwargs):
                                                          requesting_user=request.user,
                                                          **kwargs)
     except utils.ResourceFileSizeException as ex:
-        context = {'file_size_error': ex.message}
-        return render_to_response('pages/create-resource.html', context,
-                                  context_instance=RequestContext(request))
+        ajax_response_data['message'] = ex.message
+        return JsonResponse(ajax_response_data)
 
     except utils.ResourceFileValidationException as ex:
-        context = {'validation_error': ex.message}
-        return render_to_response('pages/create-resource.html', context,
-                                  context_instance=RequestContext(request))
+        ajax_response_data['message'] = ex.message
+        return JsonResponse(ajax_response_data)
 
     except Exception as ex:
-        context = {'resource_creation_error': ex.message}
-        return render_to_response('pages/create-resource.html', context,
-                                  context_instance=RequestContext(request))
-
+        ajax_response_data['message'] = ex.message
+        return JsonResponse(ajax_response_data)
+    # TODO: (Pabitra) Not sure how we should be handling this when the call now comes as an
+    # ajax call - this is relevant to only reference time series resource type
     if url_key in page_url_dict:
         return render(request, page_url_dict[url_key], {'title': res_title, 'metadata': metadata})
```

Comment:
@pkdash I believe this needs to be taken care of by the ajax call itself, doing a redirect or something, but not from the view function itself now that it is been changed to an AJAX call. @zhiyuli @jsadler2 need to weigh in here to see whether reference time series will be breaking with this change. @zhiyuli @jsadler2 You may want to pull this branch to test reference time series to see if it still works. Unless reference time series has been changed not to rely on this render to a new page, I think this will break reference time series.